test/e2e/tests/send/send-erc20.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { CHAIN_IDS } from '../../../../shared/constants/network';
+import Confirmation from '../../page-objects/pages/confirmations/redesign/confirmation';
+import FixtureBuilder from '../../fixture-builder';
+import HomePage from '../../page-objects/pages/home/homepage';
+import SendPage from '../../page-objects/pages/send/send-page';
+import { loginWithoutBalanceValidation } from '../../page-objects/flows/login.flow';
+import { withFixtures } from '../../helpers';
+import { mockSendRedesignFeatureFlag } from './common';
+
+const getERC20Fixtures = () => {
+  return new FixtureBuilder()
+    .withNetworkControllerOnMainnet()
+    .withEnabledNetworks({
+      eip155: {
+        [CHAIN_IDS.MAINNET]: true,
+        [CHAIN_IDS.LINEA_MAINNET]: true,
+      },
+    })
+    .withTokensController({
+      tokenList: [],
+      tokensChainsCache: {
+        '0x1': {
+          data: {
+            '0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267': {
+              name: 'Entropy',
+              aggregators: ['Lifi', 'Coinmarketcap', 'Rango'],
+              address: '0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267',
+              decimals: 18,
+              iconUrl:
+                'https://static.cx.metamask.io/api/v1/tokenIcons/1/0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267.png',
+              occurrences: 3,
+              symbol: 'ERP',
+            },
+          },
+        },
+      },
+    })
+    .build();
+};
+
+describe('Send ERC20', function () {
+  it('it should be possible to send ERC20 token', async function () {
```

Comment:
this title doesn't match what we are doing in the spec: we are. performing steps for Sending ETH, not Sending an ERC20 token.
If we want the latter, we should deploy an ERC20 token contract in the specific network we need it, and then get the contract address and use that for sending the ERC20

Reply:
Good point @seaona, I removed ERC20 check from the PR, I will create a separate PR for it.